src/AppInstallerCLICore/Workflows/SettingsFlow.cpp

```diff
@@ -99,4 +126,22 @@ namespace AppInstaller::CLI::Workflow
             ShellExecuteW(nullptr, nullptr, L"notepad", filePathUTF16.c_str(), nullptr, SW_SHOW);
         }
     }
+
+    void ExportAdminSettings(Execution::Context& context)
+    {
+        AdminSetting settings[] =

```

Comment:
This would be better if there were a `Max` enum value and we could just enumerate `for (i = firstValidSetting; i < AdminSetting::Max; ++i)` so that any future settings are automatically included.